src/bidiMapper/domains/script/realm.ts

```diff
@@ -110,13 +110,16 @@ export class Realm {
       }
     }
 
-    if (
-      cdpValue.result.type === 'object' &&
-      cdpValue.result.subtype === 'generator'
-    ) {
-      return {
-        type: 'generator',
-      };
+    if (cdpValue.result.type === 'object') {
+      switch (cdpValue.result.subtype) {
+        case 'generator':
+        case 'iterator':
```

Comment:
Nit: Add TODO about Array/String iterator

Reply:
This code is going to remain the same though. The TODO should be tracked in the bug (or in V8).

What I could do is add an unit test that is disabled and has a TODO.

Reply:
Added tests + TODO